### PR TITLE
Migrate Cloud::ConfigurationController#index to bootstrap

### PR DESCRIPTION
**Pull request by vpereira**
Migrate the `Cloud::ConfigurationController#index` to boostrap:

Screenshot_2019-05-31_11-35-14

### CODE REVIEW

**Comment by vpereira** [comment]
[Cloud Upload Configuration](https://trello.com/c/7T3ZhHuk/1155-cloud-upload-configuration)

**Comment by vpereira** [comment]
no security issue here, the Hakiri stop simply didn't run and I don't find a way to respin it again

File: src/api/app/controllers/webui/cloud/configurations_controller.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class ConfigurationsController < WebuiController
       before_action :set_breadcrumb
 
       def index
+        switch_to_webui2
```
Comment (saraycp) [inline src/api/app/controllers/webui/cloud/configurations_controller.rb]: Please, add the condition until we finish the migration:

```
switch_to_webui2 if Rails.env.development?
```

File: src/api/app/views/webui2/webui/cloud/configurations/_breadcrumb_items.html.haml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+= render partial: 'webui/main/breadcrumb_items'
+
+%li.breadcrumb-item.active{ 'aria-current' => 'page' }
```
Comment (saraycp) [inline src/api/app/views/webui2/webui/cloud/configurations/_breadcrumb_items.html.haml]: If I'm not wrong, we don't need this `active` here, only in line 5.

Comment (vpereira) [reply src/api/app/views/webui2/webui/cloud/configurations/_breadcrumb_items.html.haml]: done

File: src/api/app/controllers/webui/cloud/configurations_controller.rb (hunk 1 above)
Comment (vpereira) [reply src/api/app/controllers/webui/cloud/configurations_controller.rb]: done

**Pull request merged**